graphql/e2e/auth/schema.graphql

```diff
@@ -848,6 +848,25 @@ type State @auth(
     country: Country
 }
 
+interface Member @auth(
+    query: { rule: "{$ROLE: { eq: \"ADMIN\" } }" },
+){
+    refID: String! @id (unique:true)
+    name:  String! @id
```

Comment:
Can you add (unique: false) to this field. It shouldn't have any effect on the behaviour. It will test out if unique arg can take false as well. Can this also be done at 1-2 places in the graphql files in schemagen/input ?